assets/src/application/ui/display/TimezoneTimeInfo/Trigger.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { forwardRef } from 'react';
+
+import { Tooltip } from '@infraUI/display';
+import { IconButton } from '@infraUI/inputs';
+
+interface Props {
+	label: string;
+	onClick?: VoidFunction;
+}
+
+const Trigger: React.FC<Props> = forwardRef(({ label, onClick }, ref) => {
+	return (
+		<Tooltip onClick={onClick} title={label}>
+			<IconButton aria-label={label} icon='info-outline' onClick={onClick} ref={ref} variant='unstyled' />
+		</Tooltip>
+	);
```

Comment:
May be add a `TODO` to replace it with `withTooltip`

Reply:
Added `TODO`, thx